jvm/src/main/scala/uk/gov/homeoffice/drt/protobuf/serialisation/FlightMessageConversion.scala

```diff
@@ -263,26 +263,43 @@ object FlightMessageConversion {
       flightMessage.totalPax.map(totalPaxSourceFromMessage).toMap ++
         Seq(TotalPaxSourceMessage(
           Option(ApiFeedSource.toString),
-          Option(PassengersMessage(actual = flightMessage.apiPaxOLD, None))
+          Option(PassengersMessage(actual = flightMessage.apiPaxOLD, getTransPaxAccordingToFeedSource(ApiFeedSource.toString, flightMessage)))
         )).map(totalPaxSourceFromMessage).toMap
     } else
       flightMessage.totalPax.map(totalPaxSourceFromMessage).toMap
 
     val includeDeprecatedActPax: Map[FeedSource, Passengers] = if (flightMessage.actPaxOLD.isDefined) {
       includeDeprecatedApiPassenger ++ Seq(TotalPaxSourceMessage(
         Option(getFeedSourceForActPax(flightMessage).toString),
-        Option(PassengersMessage(actual = flightMessage.actPaxOLD, flightMessage.tranPaxOLD))
+        Option(PassengersMessage(actual = flightMessage.actPaxOLD, getTransPaxAccordingToFeedSource(getFeedSourceForActPax(flightMessage).toString, flightMessage)))
       )).map(totalPaxSourceFromMessage).toMap
     } else includeDeprecatedApiPassenger
 
-    includeDeprecatedActPax
+    val includeDeprecatedPaxInPassenger: Map[FeedSource, Passengers] = if (flightMessage.totalPax.map(_.paxOLD).nonEmpty) {
+      includeDeprecatedActPax ++ flightMessage.totalPax.filter(_.paxOLD.isDefined).map(a =>
+        TotalPaxSourceMessage(
+          Option(a.getFeedSource),
+          Option(PassengersMessage(actual = a.paxOLD, getTransPaxAccordingToFeedSource(a.getFeedSource, flightMessage)))
+        )).map(totalPaxSourceFromMessage).toMap
+    } else includeDeprecatedActPax
+
+    includeDeprecatedPaxInPassenger
+  }
+
+  private def getTransPaxAccordingToFeedSource(searchFeedSource: String, flightMessage: FlightMessage): Option[Int] = {
```

Comment:
We need to add `ForecastFeedSource` to the sources we check

Reply:
Actually do we need this function? If we're already getting the `actPaxOLD` then I'm not sure there's any reason we wouldn't also get the `tranPaxOLD` for the same source

Reply:
I have added this bcos , transPaxOld we get either from live or Api source . I assumed here we dont get it from ForecastFeedSource